tests/pester-tests/commands/choco-upgrade.Tests.ps1

```diff
@@ -1,4 +1,4 @@
-﻿Describe "choco upgrade" -Tag Chocolatey, UpgradeCommand {
+Describe "choco upgrade" -Tag Chocolatey, UpgradeCommand {

```

Comment:
I presume the encoding on this changed. I am not entirely sure why, but will investigate it prior to merging.

Reply:
Not sure why it changed the encoding, but it's been changed back.